_posts/2023-02-22-user-defined-operators.md

```diff
@@ -0,0 +1,484 @@
+---
+layout: post
+published: true
+title: 'Feature Deep-Dive: User-Defined Operators'
+date: '2023-02-22'
+author: Kamil Śliwak, Matheus Aguiar
+category: Explainers
+---
+
+The prominent new feature in [Solidity 0.8.19](https://github.com/ethereum/solidity/releases/tag/v0.8.19)
+is the support for defining *operators* on [user defined value types](https://docs.soliditylang.org/en/latest/types.html#user-defined-value-types) (UDVTs).
+
+If you have not been keeping up with recent features, UDVTs are a new class of types introduced in Solidity 0.8.8.
+They provide zero-cost abstraction of an elementary value type that results in a completely new type.
+This is similar to creating an alias, but the new type is distinct from the underlying value type
+and all other UDVTs derived from it.
+The ability to use operators is meant to bring the UDVTs closer to being as natural to use as the
+built-in types and promote the use of type-safe interfaces.
+Below, we are going to take a closer look into the details of this new mechanism.
+
+### A Quick Refresher
+Before we get into the details of the new feature and how it affects the language, it may be a good
+idea to first go over the two existing features that it builds upon: built-in operators and `using for`.
```

Comment:
Can we link to the appropriate place in the docs for built-in-operators and `using-for`?

Reply:
They are already linked, at the beginning of each respective section.